README.md

```diff
@@ -311,17 +311,17 @@ Block number: 5
 There is currently only support to create a Byron update proposal:
 
 ```
-cardano-cli create-byron-update-proposal --config NODE-CONFIGURATION
```

Comment:
Would it be better to put all of the command line options at a fixed indentation so we never have this whitespace fiddling again?